tests/test_filters.py

```diff
@@ -86,6 +87,12 @@ def test_longdate_datetime_with_tz(self):
         with self.app.test_request_context('/'):
             assert filters.longdate(testdate) == testdate.strftime('%e %B %Y')
 
+    def test_date_filter(self):
+        testdate = self.today
+        testdate_string = self.today.strftime('%Y-%m-%d')
+        with self.app.test_request_context('/'):
+            assert filters.date_filter(testdate, 'yyyy-MM-dd', usertz=False) == testdate_string
```

Comment:
Test with (a) default format, (b) named formats (short, medium, long) and (c) custom format strings. Compare all with the static output string.

Then do this again with a different language passed to `test_request_context`.

Reply:
Got it. Will push shortly.